independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanDeployment.java

```diff
@@ -775,18 +778,23 @@ private Map<DotName, StereotypeInfo> findStereotypes(IndexView index, Map<DotNam
                         isNamed = true;
                     } else if (DotNames.PRIORITY.equals(annotation.name())) {
                         alternativePriority = annotation.value().asInt();
+                    } else if (DotNames.ARC_PRIORITY.equals(annotation.name()) && alternativePriority == null) {
```

Comment:
Good catch. Do we have a test for it?

Reply:
Good question, I don't think so. I'll add one.

Reply:
Actually one of the tests I added in this PR uses ArC's `@AlternativePriority`, so that is actually covered. There's indeed no test for ArC's `@Priority`.